src/platform/packages/shared/kbn-alerts-ui-shared/src/alert_filter_controls/filter_group.tsx

```diff
@@ -222,10 +222,6 @@ export const FilterGroup = (props: PropsWithChildren<FilterGroupProps>) => {
   }, [controlsUrlState, getStoredControlState, switchToEditMode]);
 
```

Comment:
The useEffect can be removed. There is no reason to cleanup subscription in multiple locations

```
useEffect(() => {
    return () => {
      [filterChangedSubscription.current, inputChangedSubscription.current].forEach((sub) => {
        if (sub) sub.unsubscribe();
      });
    };
  }, []);
```